src/js/Tabs.js

```diff
@@ -214,25 +216,32 @@ function Tabs(rootEl, config) {
 	init();
 }
 
-Tabs.init = function(el, config) {
-	const tabs = [];
-	let tEls;
-	let c;
-	let l;
-	if (!el) {
-		el = document.body;
-	} else if (!(el instanceof HTMLElement)) {
-		el = document.querySelector(el);
-	}
-	if (el.querySelectorAll) {
-		tEls = el.querySelectorAll('[data-o-component=o-tabs]');
-		for (c = 0, l = tEls.length; c < l; c++) {
-			if (!tEls[c].matches('[data-o-tabs-autoconstruct=false]') && !tEls[c].hasAttribute('data-o-tabs--js')) {
-				tabs.push(new Tabs(tEls[c], config));
-			}
+Tabs.init = function(rootEl, config) {
+	if (!rootEl) {
+		rootEl = document.body;
+	}
+	if (!(rootEl instanceof HTMLElement)) {
+		rootEl = document.querySelector(rootEl);
+	}
+
+	if (rootEl instanceof HTMLElement && /\bo-tabs\b/.test(rootEl.getAttribute('data-o-component'))) {
+		if (!rootEl.matches('[data-o-tabs-autoconstruct=false]') && !rootEl.hasAttribute('data-o-tabs--js')) {
```

Comment:
Just looking at this again with fresh eyes, if a consumer is specifying `Tabs.init('#o-tabs-element')` then they probably want this to init and the `autoconstruct` option shouldn't be a factor here? What do you think @JakeChampion @rowanmanning ?